packages/front-end/components/Layout/AccountPlanNotices.tsx

```diff
@@ -89,6 +89,22 @@ export default function AccountPlanNotices() {
   if (!isCloud() && license) {
     // Trial license is up
     const licenseTrialRemaining = license.trial ? daysLeft(license.exp) : -1;
+    if (license?.org && license.org !== organization.id) {
+      return (
+        <Tooltip
+          body={
+            <>
+              Your license key appears to be invalid. Please contact
+              sales@growthbook.io for assistance.
```

Comment:
Does it make sense to make this a clickable email link?

Reply:
Unfortunately not because the tooltip currently disappears when not hovering over the original item.